docs/wiki/configurator/failsafe-tab.md

```diff
@@ -4,8 +4,125 @@ sidebar_position: 5
 
 # Failsafe Tab
 
+Failsafe is the action your craft takes when it loses connection to the transmitter.
+This tab allows you to configure what your craft does in an event of a failsafe.
+
+:::danger
+It is very important to set up your failsafe correctly to prevent your craft from flying away or crashing when you lose connection.
+:::
+
+![Failsafe tab](/img/betaflight_configurator_failsafe_tab.png)
+
+## Failsafe Stages
+
+There are two stages of failsafe:
+
+- **Stage 1** - entered when a flight channel has an invalid pulse length, the receiver reports failsafe mode or
+  there is no signal from the receiver at all, the channel fallback settings are applied to all channels and
+  a short amount of time is provided to allow for recovery.
+
+- **Stage 2** - entered when the error condition takes longer than the configured guard time while the craft is armed,
+  all channels will remain at the applied channel fallback setting unless overruled by the chosen procedure.
+
```

Comment:
Channels will remain...

Reply:
@ctzsnooze  What do you mean here, what should I change?

Reply:
we could say
"Entered when radio link is lost for more than the configured Stage 1 duration, and the quad is armed. The Stage 2 Failsafe Mode will be activated. Channels that are not controlled by the failsafe mode remain at their fallback setting. "